journal-of-clinical-neurophysiology.csl

```diff
@@ -13,8 +13,8 @@
     </author>
     <category citation-format="numeric"/>
     <category field="biology"/>
-    <issn>0736-0258</issn>
     <eissn>1537-1603</eissn>
+    <issn>0736-0258</issn>
```

Comment:
@damnation333, any reason you did this? I had already restarted the failing Travis build.